src/Workspaces/CSharp/Portable/Extensions/ITypeSymbolExtensions.cs

```diff
@@ -54,6 +54,17 @@ internal static partial class ITypeSymbolExtensions
             return SyntaxFactory.RefType(refKeyword, underlyingType);
         }
 
+        public static TypeSyntax GenerateRefReadOnlyTypeSyntax(
+            this INamespaceOrTypeSymbol symbol)
+        {
+            var underlyingType = GenerateTypeSyntax(symbol)
```

Comment:
GenerateTypeSyntax already adds the simplifier annotation.

Reply:
Then why is it specified in `GenerateRefTypeSyntax()`?

---
In reply to: [141181527](https://github.com/dotnet/roslyn/pull/22285#discussion_r141181527) [](ancestors = 141181527)